### DISCUSSION TITLE
[Brainstorming] Proposal for the Tech Writer Role

### DISCUSSION THREAD

**Discussion by afzal442** [apache/apisix#6049] [Ideas]
🔹 Here I came up with some ideas by having discussion with @juzhiyuan and reproduced the points needed while curating any content for a blog or a video of our API7 and Apache APISIX documentation. 📝 

- Plan out the use cases for the products/tools managed and used through the Project
- Sync the Blogs, Articles etc. based on USERS and DEVELOPERS' view

Note:

🏷️ More things to be added, feel free to edit or add any suggestions in the comment below. Thanks! 😃

**Comment by tzssangglass** [comment]
lgtm

**Comment by juzhiyuan** [comment]
Hi @afzal442, Apache APISIX project is fully under the Apache Software Foundation, for our community, we have contributors, committers, PMCs, and more roles you could find on community.apache.org, but no official tech writers roles AFAIK, we're all volunteers, right?

>  Here I came up with some ideas by having discussion with @juzhiyuan and reproduced the points needed while curating any content for a blog or a video of our API7 and Apache APISIX documentation. 📝

Actually, we talked about `contents around Apache APISIX`, and API7.ai is a company that I'm in, there has no relation between them.

> Plan out the use cases for the products/tools managed and used through the Project
> Sync the Blogs, Articles etc. based on USERS and DEVELOPERS' view

Back to your proposal, use cases, practices, or documentation is important for newcomers or developers, you could choose one specific to work on :)

**Reply by afzal442** [reply]
Awesome! Thanks @juzhiyuan! Looking forward to seeing next as a volunteer.
